tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "esModuleInterop": true,
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["ES2023"],
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
```

Comment:
Maybe we should use `Node16` here to avoid automatically introducing upstream changes without review.

```suggestion
    "module": "Node16",
    "moduleResolution": "Node16",
```

Reply:
I set it to `NodeNext` to support `import.meta.dirname`, but had to change that later anyway. Updated to `Node16`.